sympy/tensor/indexed.py

```diff
@@ -143,19 +42,36 @@ def __new__(cls, base, *args, **kw_args):
             base = IndexedBase(base)
         elif not isinstance(base, IndexedBase):
             raise TypeError(filldedent("""
-                Indexed expects string, Symbol or IndexedBase as base."""))
+                Indexed expects string, Symbol, or IndexedBase as base."""))
         args = list(map(sympify, args))
         return Expr.__new__(cls, base, *args, **kw_args)
 
     @property
+    def _diff_wrt(self):
+        """Allow derivatives with respect to an ``Indexed`` object."""
+        return True
+
+    def _eval_derivative(self, wrt):
+        if isinstance(wrt, Indexed) and wrt.base == self.base:
+            if len(self.indices) != len(wrt.indices):
+                msg = "Different # of indices: d({!s})/d({!s})".format(self,
+                                                                       wrt)
+                raise IndexException(msg)
+            result = S.One
+            for index1, index2 in zip(self.indices, wrt.indices):
+                result *= DeltaIndexedBase()[index1, index2]
+            return result
+        else:
+            return S.Zero
+
+    @property
     def base(self):
-        """Returns the IndexedBase of the Indexed object.
+        """Returns the ``IndexedBase`` of the ``Indexed`` object.
```

Comment:
Ah sorry, I confused that with GitHub markup, where a single backtick is indeed enough.